landscape/client/manager/shutdownmanager.py

```diff
@@ -58,9 +58,31 @@ def _respond_success(self, data, operation_id):
             lambda _: self.registry.broker.stop_exchanger())
         return deferred
 
-    def _respond_failure(self, failure, operation_id):
+    def _respond_failure(self, failure, operation_id, reboot):
         logging.info("Shutdown request failed.")
-        return self._respond(FAILED, failure.value.data, operation_id)
+        failure_report = '\n'.join([
+            failure.value.data,
+            "",
+            "Attempting to force %(operation)s. Please note that if this "
+            "succeeds, Landscape will have no way of knowing and will still "
+            "mark this activity as having failed. It is recommended you check "
+            "the state of the machine manually to determine whether "
+            "%(operation)s succeeded." %
+            {"operation": "reboot" if reboot else "shutdown"}
+        ])
```

Comment:
IMHO, a "{}\n\nBlah blah {0}...".format(operation=...)
would be mildly more readable than this join and % format, but feel free to disregard my opinion.

Reply:
Sure, will replace with .format